triaging.rst

```diff
@@ -9,6 +9,41 @@ and developers.
 Contributors with the Developer role on the `issue tracker`_ can triage issues
 directly without any assistance.
 
+Additionally, this section provides an overview of the Python triage team.
+
+Python triage team
+-----------------------------
```

Comment:
Please shorten underline to match heading.

Reply:
@willingc What is the length requirement of the underline in comparison to the title length in reST? When the title length was increased, I received build errors due to the previous one being too short.

Reply:
Based on the other title underlines, it looks the the end of the title must be on the same character position in which the last word of the header starts (at minimum). Let me know whether or not this is acceptable.

Reply:
The underline should be the same as the line above it.

```
Python triage team
-------------------
```

Reply:
Oh my apologies, for some reason I was basing it on the visual length, not the actual number of characters. Fixed.